Controller/ConnectController.php

```diff
@@ -359,4 +348,60 @@ protected function getTemplatingEngine()
     {
         return $this->container->getParameter('hwi_oauth.templating.engine');
     }
+
+    /**
+     * {@inheritdoc}
+     *
+     * Symfony <2.6 BC. To be removed.
+     */
+    protected function isGranted($attributes, $object = null)
```

Comment:
I'm not fan of such things, looks ugly and adds overhead cause of all those `method_exists()` calls without any real benefit...